prov/gni/src/gnix_dom.c

```diff
@@ -128,8 +128,8 @@ static int gnix_domain_close(fid_t fid)
 	references_held = _gnix_ref_put(domain);
 
 	if (references_held) {
-		GNIX_WARN(FI_LOG_DOMAIN, "failed to fully close domain due to "
-				"lingering references. references=%i dom=%p\n",
+		GNIX_INFO(FI_LOG_DOMAIN, "failed to fully close domain due to "
+			  "lingering references. references=%i dom=%p\n",
 			  references_held, domain);
```

Comment:
Other similar ones like for endpoints and cq use GNIX_INFO, so I was making this consistent.  We could set them all to warn, but then you'd still get the warn if FI_LOG_LEVEL=info.